File: src/main/java/pl/training/workshop/m6/s09_observer/step1/ReservationPaid.java
package pl.training.workshop.m6.s09_observer.step1;

import pl.training.workshop.shared.Money;

/** Krok 1: zdarzenie - fakt "rezerwacja opłacona" z danymi potrzebnymi odbiorcom. */
public record ReservationPaid(String reservationId, String email, String phone, Money amount) {
}

File: csharp/src/Training.Workshop/M6/S09Observer/Step1/ReservationPaid.cs
using Training.Workshop.Shared;

namespace Training.Workshop.M6.S09Observer.Step1;

/// <summary>Krok 1: zdarzenie - fakt "rezerwacja opłacona" z danymi potrzebnymi odbiorcom.</summary>
public sealed record ReservationPaid(string ReservationId, string Email, string Phone, Money Amount);
